assets/app/views/_template.html

```diff
@@ -0,0 +1,7 @@
+<li class="template">
+  <img ng-src="{{template.icon}}">
```

Comment:
It will not be always defined. We need a default image. Was thinking we could add it to the template object in the controller before displaying it